packages/create-onchain/templates/minikit-basic/app/components/DemoComponents.tsx

```diff
@@ -386,33 +381,39 @@ function TodoList() {
   );
 }
 
-
 function TransactionCard() {
   const { address } = useAccount();
 
   // Example transaction call - sending 0 ETH to self
-  const calls = useMemo(() => address
-    ? [
-        {
-          to: address,
-          data: "0x" as `0x${string}`,
-          value: BigInt(0),
-        },
-      ]
-    : [], [address]);
+  const calls = useMemo(
+    () =>
+      address
+        ? [
+            {
+              to: address,
+              data: "0x" as `0x${string}`,
```

Comment:
Just musing, but I'm curious why this wasn't originally written as "0x0" or something that fits the type, instead of casting to the type